### PR TITLE
update azure_rm_aks document--02

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_aks relate module document

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_aks/azure_rm_aks_facts/azure_rm_aksversion_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_aksversion_facts.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 version_added: "2.8"
 
-short_description: Get available kubernetes versions supported by Azure Kubernetes Service.
+short_description: Get available kubernetes versions supported by Azure Kubernetes Service
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_aksversion_facts.py:21]: why remove .?

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_aksversion_facts.py:21]: Here is a little hint from Ansible document, short_description should not have a trailing period/full stop([Link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)). Thanks a lot!

**Review by yungezz — APPROVED** [review]

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

**Pull request merged**